local-cli/install/install.js

```diff
@@ -10,7 +11,17 @@ log.heading = 'rnpm-install';
 function install(args, config) {
   const name = args[0];
 
-  var res = spawnSync('npm', ['install', name, '--save'], spawnOpts);
+  const projectDir = config.getProjectRoots()[0];
+  const isYarnAvailable =
+    yarn.getYarnVersionIfAvailable() &&
+    yarn.isGlobalCliUsingYarn(projectDir);
+
+  var res;
+  if (isYarnAvailable) {
+    res = spawnSync('yarn', ['add', name], spawnOpts);
+  } else {
+    res = spawnSync('npm', ['install', name, '--save'], spawnOpts);
```

Comment:
Aren't we like including that piece of functionality in many places? Wouldn't it be easier to just have a `spawnSync` util, called `callYarnOrNpm` that abstracts that? There are few other places where its going to be needed and so I'd like to keep it as minimal as possible.